frontend/server/src/main/java/org/pytorch/serve/util/ConnectorType.java

```diff
@@ -4,5 +4,6 @@ public enum ConnectorType {
     INFERENCE_CONNECTOR,
     MANAGEMENT_CONNECTOR,
     METRICS_CONNECTOR,
-    ALL
+    ALL,
```

Comment:
When "ALL" is set, both INFERENCE_CONNECTOR and MANAGEMENT_CONNECTOR will take effective. What's the expectation when OPEN_INFERENCE_CONNECTOR is added?